packages/odyssey-react-mui/src/properties/translations/odyssey-react-mui_cs.properties

```diff
@@ -14,5 +14,22 @@ table.fetchedrows.text = Byl na\u010Dten {{totalRows}} \u0159\u00E1dek
 table.fetchedrows.text_plural = Celkem na\u010Dteno \u0159\u00E1dk\u016F: {{totalRows}}
 table.rows.text = {{totalRows}} \u0159\u00E1dek
 table.rows.text_plural = Po\u010Det \u0159\u00E1dk\u016F: {{totalRows}}
+table.actions = [[odyssey:odyssey-react-mui: table.actions]]
+table.draghandle.tooltip = [[odyssey:odyssey-react-mui: table.draghandle.tooltip]]
+table.draghandle.arialabel = [[odyssey:odyssey-react-mui: table.draghandle.arialabel]]
+table.reorder.tofront = [[odyssey:odyssey-react-mui: table.reorder.tofront]]
+table.reorder.forward = [[odyssey:odyssey-react-mui: table.reorder.forward]]
+table.reorder.backward = [[odyssey:odyssey-react-mui: table.reorder.backward]]
+table.reorder.toback = [[odyssey:odyssey-react-mui: table.reorder.toback]]
+table.density.arialabel = [[odyssey:odyssey-react-mui: table.density.arialabel]]
+table.columnvisibility.arialabel = [[odyssey:odyssey-react-mui: table.columnvisibility.arialabel]]
+table.moreactions.arialabel = [[odyssey:odyssey-react-mui: table.moreactions.arialabel]]
+filters.filters.arialabel = [[odyssey:odyssey-react-mui: filters.filters.arialabel]]
+filters.filter.any = [[odyssey:odyssey-react-mui: filters.filter.any]]
+filters.filter.clear = [[odyssey:odyssey-react-mui: filters.filter.clear]]
+filters.menuitem.selected = [[odyssey:odyssey-react-mui: filters.menuitem.selected]]
+filters.menuitem.any = [[odyssey:odyssey-react-mui: filters.menuitem.any]]
+filters.clear.label = [[odyssey:odyssey-react-mui: filters.clear.label]]
+filters.search.label = [[odyssey:odyssey-react-mui: filters.search.label]]
```

Comment:
I'm assuming these values and the ones like it in other properties files will get replaced with translations eventually? Sorry for the question about this if it's common knowledge... I just haven't seen this before.